mesonbuild/interpreter/kwargs.py

```diff
@@ -393,6 +393,7 @@ class Executable(_BuildTarget):
     pie: T.Optional[bool]
     vs_module_defs: T.Optional[T.Union[str, File, build.CustomTarget, build.CustomTargetIndex]]
     win_subsystem: T.Optional[str]
+    android_exe_type: T.Optional[str]
```

Comment:
```suggestion
    android_exe_type: T.Optional[Literal['application', 'executable']]
```

This can later propagate down to help catch typos